DOCS-2099: Adjusts content on Community Integration page (#1132)
MultiSafepay/docs: content/integrations/community/Community integration requirements/_index.md

aliases: [/integrations/community/community/]
---

> At MultiSafepay, our developers write code to process in-house transactions. However, there are also many fantastic developers worldwide who create code wrappers, plugins and applications in order to work with us.

Reviewer: ```suggestion
At MultiSafepay, our developers write code to process in-house transactions. However, there are also many fantastic developers worldwide who create code wrappers, plugins, and applications in order to integrate our products and services.
```

> We are happy to accompany you starting with a place on our documentation pages as well as assist you on marketing level. If you would like for your project to become a part of MultiSafepay DOCS, please contact the MultiSafepay Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
We are happy to accompany you, starting with a place on our documentation pages as well as assist you on a marketing level. If you would like for your project to become a part of MultiSafepay Docs, please contact the MultiSafepay Integration Team at <integration@multisafepay.com>
```

> We would like to outline the main requirements that you will have to adhere to if you would like to be a part of the MultiSafepay community.

Reviewer: ```suggestion
If you want to publish a MultiSafepay community integration, you have to adhere to the following agreements:
```

> The agreements are the following:

Reviewer: ```suggestion
```
